src/graph/bipartite.cc

```diff
@@ -0,0 +1,824 @@
+/*!
+ *  Copyright (c) 2019 by Contributors
+ * \file graph/bipartite.cc
+ * \brief Bipartite graph implementation
+ */
+#include <dgl/array.h>
+#include <dgl/lazy.h>
+#include <dgl/immutable_graph.h>
+
+#include "./bipartite.h"
+#include "../c_api_common.h"
+
+namespace dgl {
+namespace {
+inline GraphPtr CreateBipartiteMetaGraph() {
+  IdArray row = aten::VecToIdArray(std::vector<int64_t>({Bipartite::kSrcVType}));
```

Comment:
Changing `int64_t` to `uint64_t` or `dgl_type_t` gets rid of the bug.

Longer story:
When I separated `std::vector<int64_t>({Bipartite::kSrcVType})` into an assignment statement:
```c++
std::vector<int64_t> row_vec = {Bipartite::kSrcVType};
```
MSVC throws a compilation error saying it requires a narrowing conversion.  However, when embedding this faulty assignment statement into an expression like you did here, MSVC somehow ignores it and outputs a vector with size 0 (i.e. generating wrong code).

Issue verified with the following code:
```c++
int size(const std::vector<int64_t> &v)
{
	return v.size();
}

int main()
{
	std::vector<int64_t> v({ (uint64_t)0 });
	std::cout << size(v) << std::endl;  // prints 0
	return 0;
}
```

I will find some time to search or raise the issue in MSDN.  Meanwhile, please just change `int64_t` to `uint64_t` or `dgl_id_t` here.